### ISSUE TITLE
Write boilerplate that will be printed out everytime

### ISSUE THREAD

**Issue by mandel01** [mjmlab/labcalc#4]
This can go in a new function in `utils.py` -- we'll use this script to hold utility functions we want to call from other functions.

Examples...

```python
def format_for_user(output):
    "print the program output in a report for the user"
    pass
```
Can also use this file for things like table formatting, etc....

```python
def format_table(lol):
    "output list of lists to tabular text format"
    pass
```

**Comment by dtarnowski16** [comment]
To make sure I understand, this script is just a place to store utility functions we are working on. We'd then need to copy in the ones we want into the actual program scripts and edit them to have the variables we want. This script would be a place to edit/explore things like formatting?

**Comment by mandel01** [comment]
No, I think that this will hold functions that multiple programs will use.

There may not be as many of those as I had thought. Each program may need to format its own output unless we can come up with a clever way to do it.

Some things may be generally useful and we could write one function that could get reused (e.g., formatting a list of lists into a text only table for printing).
